Disable the documentation build

The documentation tool is compiled for the target, but executed on the
host. We don't bother fixing this, as we don't care of the
documentation in Buildroot.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: mutt-1.5.17+20080114/Makefile.am
===================================================================
--- mutt-1.5.17+20080114.orig/Makefile.am	2010-05-15 18:15:08.000000000 +0200
+++ mutt-1.5.17+20080114/Makefile.am	2010-05-15 18:15:17.000000000 +0200
@@ -9,7 +9,7 @@
 IMAP_INCLUDES = -I$(top_srcdir)/imap
 endif
 
-SUBDIRS = m4 po intl doc contrib $(IMAP_SUBDIR)
+SUBDIRS = m4 po intl contrib $(IMAP_SUBDIR)
 
 bin_SCRIPTS = muttbug flea @SMIMEAUX_TARGET@
